packages/eth/src/websocket/connection.rs

```diff
@@ -60,6 +61,10 @@ impl EthApi for WsConnection {
         Ok(self.provider.get_balance(address, None).await?)
     }
 
+    fn commit_interval(&self) -> Result<NonZeroU32> {
```

Comment:
We can remove the `Result` as this is cached upon connecting and will always be valid if the object was created.